frontend/components/ticket/EditShareWith.vue

```diff
@@ -81,25 +83,39 @@
 
           }
           this.username = "";
-          this.query = ""
+          this.query = "";
+          this.filterGuests()
         }).catch(error => {
               this.usernameErrors = error.response.data
             }
         )
-
       },
       submit(user) {
         this.username = user.username;
         this.query = user.first_name + " " + user.last_name;
         this.open = false
       },
       filterGuests() {
-        axios.get("/api/inboxes/" + this.inbox_id + "/guests", {params: {"q": this.query, "size": 5}}).then(response => {
+        axios.get("/api/inboxes/" + this.inbox_id + "/guests", {
+          params: {
+            "q": this.query,
+            "size": 5
+          }
+        }).then(response => {
           this.guests = response.data;
+          this.removeAuthorFromGuests()
         })
       },
       away() {
         this.open = false
+      },
+      removeAuthorFromGuests() {
+        // Remove author from list
+        if (this.author) {
+          this.guests = this.guests.filter(obj => {
+            return obj.id !== this.author.id
+          })
+        }
```

Comment:
Lijkt mij handiger dat dit gebeurd in een computed

Reply:
Of doe het op de backend

Reply:
De backend wordt een complexe aanpassing, het heeft geen impact als de auteur wel als shared toegevoegd wordt